### ISSUE TITLE
NTR: T-SPOT-Tuberculosis test positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8093]
New term request T-SPOT-Tuberculosis test positive (seidel_190313153624): QuantiFERON-TB Gold; interferon-gamma release assays; IGRA; TB; TSPOT

**Comment by pnrobinson** [comment]
@MickeySegal -- to confirm -- this is performed in blood?

**Comment by pnrobinson** [comment]
@MickeySegal 
Note primary label is not manufacturer specific

Adding new term: 
- ID: HP:6000901
- label: Positive bloodstream tuberculosis-induced T-cell-based interferon-gamma release assay
- parents: [Positive blood infectious agent test (HP:0430150)](https://hpo.jax.org/app/browse/term/HP:0430150)
- pmid: PMID:28798488
- def: Detection of Mycobacterium tuberculosis in the bloodstream by means of detection of interferon-gamma secreted by T cells in response to specific antigens from Mycobacterium tuberculosis, commonly 6kD early secretory antigenic target-6 and 10kD culture filtrate protein-10.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8093
- synonyms: Positive T-SPOT-tuberculosis test in the bloodstream(NONE)
